src/main/java/bio/terra/janitor/service/iam/IamService.java

```diff
@@ -0,0 +1,33 @@
+package bio.terra.janitor.service.iam;
+
+import bio.terra.janitor.app.configuration.IamConfiguration;
+import bio.terra.janitor.common.exception.UnauthorizedException;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+
+/**
+ * IAM Service which handles authorization. Currently, Janitor uses a simple user email allow list
+ * to check user permission.
+ *
+ * <p>TODO(PF-81): Switch to use SAM when GKE SAM is ready to use.
+ */
+@Component
+public class IamService {
+  private final IamConfiguration iamConfiguration;
+
+  @Autowired
+  public IamService(IamConfiguration iamConfiguration) {
+    this.iamConfiguration = iamConfiguration;
+  }
+
+  /** Check if user is an administrator. Throws {@link UnauthorizedException} if not. */
+  public void isAdminUser(AuthenticatedUserRequest userReq) {
+    if (iamConfiguration.isConfigBasedAuthZEnabled()) {
+      return;
+    }
+    boolean isAdmin = iamConfiguration.getAdminUsers().contains(userReq.getEmail());
+    if (!isAdmin)
```

Comment:
style nit: has there already been a discussion about adhering to Google's Java style guide? They prefer braces everywhere https://google.github.io/styleguide/javaguide.html#s4.1.1-braces-always-used

(That said, my attitude is that we generally shouldn't worry at all about stuff like this, and we should aim to add an auto-formatter sooner than later to make things reliably consistent w/ a known style guide. AoU used Spotless, which worked really well for Java code. I can set that up for MCT repos.)

Reply:
Some of my intellij auto-complete not adding braces, just manually added it back. 
Janitor also uses Spotless, but I don't think it can add braces like that. 
https://github.com/DataBiosphere/terra-resource-janitor/blob/master/build.gradle#L80